1_fetch/src/get_nwis_data.R

```diff
@@ -16,33 +15,28 @@ download_nwis_data <- function(site_nums = c("01427207", "01432160", "01436690",
   return(data_out)
 }
 
-nwis_site_info <- function(fileout, site_data){
-  site_no <- unique(site_data$site_no)
-  site_info <- dataRetrieval::readNWISsite(site_no)
+nwis_site_info <- function(site_data, fileout){
+  site_data<- read_csv(site_data)
```

Comment:
I would suggest renaming your argument here so that it is clear `site_data` the argument is a filename and then is read in here on L19 to become a data.frame. Something like `site_data_file` or `filein` to match your `fileout` convention.